src/cli_callback.c

```diff
@@ -751,17 +587,6 @@ void get_file_id_callback(uv_work_t *work_req, int status)
     cli_api->last_cmd_req = cli_api->curr_cmd_req;
     cli_api->rcvd_cmd_resp = "get-file-id-resp";
 
-    if (req->status_code == 401) {
```

Comment:
no need to check the error code?

Reply:
Not in this case. The "get .. id " functions don't do anything anymore, they are synchronous and cannot fail (unless they're unable to allocate memory).